### ISSUE TITLE
The technical details of EMICA?

### ISSUE THREAD

**Issue by weihaosky** [radekd91/inferno#30]
Thanks for this excellent work!
In the readme, it says EMICA is a combination of DECA, EMOCA, SPECTRE, and MICA, but without any details.
May I ask how do you combine these methods to propose EMICA? Is there any documents (e.g., paper, technical report) introducing the EMICA?
Many thanks!

**Comment by radekd91** [comment]
In short, MICA's prediction for shape, another encoder that predicts camera and pose, and one more encoder that predicts the expression, that one is supervised with EMOCA's and SPECTRE's loss. 

There is no paper or technical report. It's just a bit of engineering combining existing papers together. You can use it through the FaceReconstruction demos. If interested, you can step through the code.

**Comment by weihaosky** [comment]
Thanks for explaining! Did you train the encoder that predicts the expression from scratch?

**Comment by stas-polukeev** [comment]
@radekd91 Hey, thanks for the answer!
I wanted to clarify: If emoca's speed is limited by the forward passes of CNN, it can be sped up to somewhat real-time. However, MICA is optimization based approach, which is fundamentally offline. Is EMICA offline or online in that regard?

**Comment by radekd91** [comment]
MICA's identity prediction is not an optimization step but a simple forward
pass through a net. An image is fed to an arcface face recognition net to
get a feature. That one is then passed though an MLP to get flame identity.

**Comment by stas-polukeev** [comment]
@radekd91 Okay, thank you for a swift response!
also is it compatible with FLAME2023 model or is it based on 2020?

**Comment by radekd91** [comment]
If you downloaded all provided face reconstruction models, you should have a 2023 version in the assets. It's called `EMICA-CVT_flame2023`

**Comment by radekd91** [comment]
> Thanks for explaining! Did you train the encoder that predicts the expression from scratch?

Correct. All other encoders (expect for MICA) were trained from scratch.
